packages/costa/src/runnable.ts

```diff
@@ -176,7 +176,7 @@ export class PluginRunnable {
       return;
     }
     this.canceled = true;
-    this.send(PluginOperator.WRITE, { event: 'destroy' });
+    this.handle.kill('SIGHUP');
```

Comment:
SIGTERM?

Reply:
And shall we have a retry with the signal when detecting the costa process is busy?

Reply:
Is there any case we need to wait the client process ends itself? If not , signal will be more quick and directly.

Reply:
If we need to wait something done, I think we could handle the signal in client and do the ends, it seems more elegantly.